src/plotly/plotly_sugar.nim

```diff
@@ -3,6 +3,19 @@ import sugar
 import sequtils
 import chroma
 
+proc newPlot*(xlabel = "", ylabel = "", title = ""): Plot[float64] =
+  ## create a plot with sane default layout.
+  result = Plot[float64]()
+  result.traces = newSeq[Trace[float64]]()
+  result.layout = Layout(title: title, width: 600, height: 600,
+                         xaxis: Axis(title: xlabel),
+                         yaxis: Axis(title: ylabel),
+                         autosize: false)
+
+proc add*[T](p: Plot[T], d: Trace[T]) =
```

Comment:
can we remove this? the API already has a lot of surface area. If we keep adding little conveniences like this, it'll be at the expense of being able to quickly grok the usage.

Reply:
Perfectly happy to remove that. I wasn't even aware of its existence to be honest, when I realized it was still using a nil check.

Reply:
ok. that was probably me. :(